.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - run: pip3 install -r requirements.txt
+      - run: ./build.sh
```

Comment:
We probably do not need the `build.py` script to run anymore because those generated themes are not used.

Reply:
Good thought.  Can I remove all the Python files with your v4 changes?

Reply:
You can remove all the theme related things.

I think you have stuff for the readme and changelog, those should probably stay. They can eventually be migrated to the gradle build.

Reply:
Wait! No, I forgot that we still need the one-dark.xml to be in the classpath.

I removed all of the other themes from the script before hand. Never mind :)